spec/services/participants/defer/mentor_spec.rb

```diff
@@ -94,5 +94,13 @@
         expect { subject.call }.to raise_error(ActionController::ParameterMissing).and not_change { profile.reload.training_status }
       end
     end
+
+    context "with incorrect course" do
+      let!(:profile) { create(:ect, lead_provider:) }
```

Comment:
I find it better to name things according to their type following the principle of least surprise, there are a bunch of tests where the `profile` is actually a `User` instance:
```suggestion
      let!(:participant_profile) { create(:ect, lead_provider:) }
```